airflow/providers/google/cloud/hooks/compute_ssh.py

```diff
@@ -225,40 +227,58 @@ def get_conn(self) -> paramiko.SSHClient:
             hostname = self.hostname
 
         privkey, pubkey = self._generate_ssh_key(self.user)
-        if self.use_oslogin:
-            user = self._authorize_os_login(pubkey)
-        else:
-            user = self.user
-            self._authorize_compute_engine_instance_metadata(pubkey)
-
-        proxy_command = None
-        if self.use_iap_tunnel:
-            proxy_command_args = [
-                "gcloud",
-                "compute",
-                "start-iap-tunnel",
-                str(self.instance_name),
-                "22",
-                "--listen-on-stdin",
-                f"--project={self.project_id}",
-                f"--zone={self.zone}",
-                "--verbosity=warning",
-            ]
-            proxy_command = " ".join(shlex.quote(arg) for arg in proxy_command_args)
-
-        sshclient = self._connect_to_instance(user, hostname, privkey, proxy_command)
+
+        max_retries = 10
```

Comment:
Should we make this parameterizable? If a user creates many ssh connections at the same time 10 might not be enough.

Reply:
Done!